src/utils.rs

```diff
@@ -227,13 +222,26 @@ pub fn ldtk_pixel_coords_to_translation_pivoted(
     entity_size: IVec2,
     pivot: Vec2,
 ) -> Vec2 {
-    let pivot_point = ldtk_coord_conversion(ldtk_coords, ldtk_pixel_height).as_vec2();
+    // Find entity's LDtk-defined position.
+    let translation = ldtk_coord_conversion(ldtk_coords, ldtk_pixel_height).as_vec2();
 
+    // Adjust entity pivot so that 0.5 is the center.
     let adjusted_pivot = Vec2::new(0.5 - pivot.x, pivot.y - 0.5);
 
+    // Find the spot on the entity which the pivot corresponds to.
     let offset = entity_size.as_vec2() * adjusted_pivot;
 
-    pivot_point + offset
+    translation + offset
+}
+
+pub fn entity_center(translation: Vec3, entity_size: IVec2, pivot: Vec2) -> Vec3 {
```

Comment:
Could you leave a brief doc for this? Something like...
```suggestion
/// Calculate the "visual center" of an  LDtk object using its current bevy translation, size, and pivot value.
pub fn entity_center(translation: Vec3, entity_size: IVec2, pivot: Vec2) -> Vec3 {
```